src/Aquifer.Jobs/Subscribers/TrackResourceContentRequestMessageSubscriber.cs

```diff
@@ -77,19 +96,11 @@ private async Task LookupIpAddressAsync(TrackResourceContentRequestMessage track
                         Country = ipData.Country
                     };
 
-                    try
-                    {
-                        await dbContext.IpAddressData.AddAsync(newRecord, ct);
-                        await dbContext.SaveChangesAsync(ct);
-                    }
-                    catch (DbUpdateException)
```

Comment:
It should no longer be possible to have duplicate key exceptions with this change.